### PR TITLE
6.6.5: Event and entity search limits in web UI

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
Documents the event and entity search limits in the web UI, starting in 6.6.3, in both the web UI search page and the 6.6.3 release notes.

## Motivation and Context
Issue: https://github.com/sensu/sensu-enterprise-go/issues/2012
PR: https://github.com/sensu/sensu-enterprise-go/pull/2018

### CODE REVIEW

File: content/sensu-go/6.6/web-ui/search.md [hunk 1]
```diff
@@ -18,6 +18,29 @@ When you apply a search to a web UI page, it creates a unique link for the page
...
+In Sensu Go 6.6.3 and subsequent versions, web UI search queries on the events and entities pages will stop after returning a certain number of matches.
+Without these limits, the search operation can diminish cluster health.
+
+{{% notice note %}}
+**NOTE**: The search limits on events and entities do not apply to Sensu Go versions prior to 6.6.3.
+{{% /notice %}}
+
+### Events search limit
+
+On the events page, if you use etcd for event storage, search queries will stop after returning a certain number of events:
```
Comment (jamesdphillips) [inline content/sensu-go/6.6/web-ui/search.md:34]: maybe we can move "if you use etcd for event storage" above, as it relates to both subheadings

File: content/sensu-go/6.6/web-ui/search.md [hunk 2]
```diff
@@ -18,6 +18,29 @@ When you apply a search to a web UI page, it creates a unique link for the page
 You can bookmark these links and share your favorite search combinations.
 You can also [save your favorite searches][8].
 
+## Events and entities search limits
+
+In Sensu Go 6.6.3 and subsequent versions, web UI search queries on the events and entities pages will stop after returning a certain number of matches.
```
Comment (hillaryfraley) [inline content/sensu-go/6.6/web-ui/search.md]:
```suggestion
In Sensu Go 6.6.3 and subsequent versions, if you use etcd for event storage, web UI search queries on the events and entities pages will stop after returning a certain number of matches.
```

File: content/sensu-go/6.6/web-ui/search.md (hunk 1 above)
Comment (hillaryfraley) [inline content/sensu-go/6.6/web-ui/search.md:32]:
```suggestion
On the events page, search queries will stop after returning a certain number of events:
```

File: content/sensu-go/6.6/web-ui/search.md [hunk 3]
```diff
@@ -18,6 +18,29 @@ When you apply a search to a web UI page, it creates a unique link for the page
...
+On the events page, if you use etcd for event storage, search queries will stop after returning a certain number of events:
+
+- In Sensu Go 6.6.3 and 6.6.4, search queries will return a maximum of 25,000 events.
+- In Sensu Go 6.6.5, seach queries will return a maximum of 50,000 events.
+
+For example, in Sensu Go 6.6.5, if you use etcd for event storage and you search in a namespace that has more than 50,000 matching events, the search results will not include matching events beyond the first 50,000.
+
+### Entities search limit
+
+Starting with Sensu Go 6.6.3, search queries on the entities page will stop after retrieving approximately 500 matches.
```
Comment (hillaryfraley) [inline content/sensu-go/6.6/web-ui/search.md]:
```suggestion
Starting with Sensu Go 6.6.3, if you use etcd for event storage, search queries on the entities page will stop after retrieving approximately 500 matches.
```

File: content/sensu-go/6.6/web-ui/search.md (hunk 1 above)
Comment (hillaryfraley) [reply content/sensu-go/6.6/web-ui/search.md:34]: I decided to add it in all three places to make sure readers see it even if they only read one section here.

File: content/sensu-go/6.6/web-ui/search.md [hunk 4]
```diff
@@ -18,6 +18,29 @@ When you apply a search to a web UI page, it creates a unique link for the page
 You can bookmark these links and share your favorite search combinations.
 You can also [save your favorite searches][8].
 
+## Events and entities search limits
+
+In Sensu Go 6.6.3 and subsequent versions, if you use etcd for event storage, web UI search queries on the events and entities pages will stop after returning a certain number of matches.
+Without these limits, the search operation can diminish cluster health.
```
Comment (hillaryfraley) [inline content/sensu-go/6.6/web-ui/search.md:24]:
```suggestion
Without these limits, the search operation can diminish cluster health.

If a web UI search reaches the limit for the events or entities page, the results count at the bottom-right corner of the page will indicate that the total number of matches exceeds the number of results listed.
```

File: content/sensu-go/6.6/web-ui/search.md [hunk 5]
```diff
@@ -18,6 +18,31 @@ When you apply a search to a web UI page, it creates a unique link for the page
...
+
+If a web UI search reaches the limit for the events or entities page, the results count at the bottom-right corner of the page will indicate that the total number of matches exceeds the number of results listed.
```
Comment (hillaryfraley) [inline content/sensu-go/6.6/web-ui/search.md]:
```suggestion
In Sensu Go 6.6.5, if a web UI search reaches the limit for the events or entities page, the results count at the bottom-right corner of the page will indicate that the total number of matches exceeds the number of results listed.
```

**Pull request merged**